modules/gason/1.0.1-0.20211018144204-c9c4aa632bd3/presubmit.yml

```diff
@@ -0,0 +1,40 @@
+matrix:
+  platform:
+    - debian10
+    - ubuntu2004
+    - macos
+    - macos_arm64
+    - windows
+  bazel:
+    - 8.x
+    - 7.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+      - "@gason//:gason"
+      - "@gason//:pretty-print"
+
+bcr_test_module:
+  module_path: ""
+  matrix:
+    platform:
+      - debian10
+      - ubuntu2004
+      - macos
+      - macos_arm64
+      - windows
+    bazel:
+      - 8.x
+      - 7.x
+  tasks:
+    run_test_module:
+      name: "Run test module"
+      platform: ${{ platform }}
+      bazel: ${{ bazel }}
+      test_targets:
+        - "@gason//:all_tests"
+      test_flags:
+        - "--enable_runfiles"
```

Comment:
I couldn't reproduce the windows issue locally, I suspect there might be a remote cache issue on Windows, can you add `--noremote_accept_cached` here to see if we can workaround it?